cmd/minikube/cmd/start.go

```diff
@@ -1042,3 +1034,30 @@ func getKubernetesVersion(old *config.ClusterConfig) string {
 	}
 	return nv
 }
+
+// maybeExitWithAdvice before exiting will try to check for different error types and provide advice
+func maybeExitWithAdvice(err error) {
+	if errors.Is(err, oci.ErrWindowsContainers) {
+		out.ErrLn("")
+		out.ErrT(out.Conflict, "Your Docker Desktop container os type is Windows but Linux is required.")
+		out.T(out.Warning, "Please change Docker settings to use Linux containers instead of Windows containers.")
+		out.T(out.Documentation, "https://minikube.sigs.k8s.io/docs/drivers/docker/#verify-docker-container-type-is-linux")
+		exit.UsageT(`You can verify your Docker container type by running:
+{{.command}}
+	`, out.V{"command": "docker info --format '{{.OSType}}'"})
+	}
+
+	if errors.Is(err, oci.ErrCPUCountLimit) {
+		out.ErrLn("")
+		out.ErrT(out.Conflict, "Your {{.name}} doesn't have enough CPUs. ", out.V{"name": viper.GetString("driver")})
```

Comment:
I would remove "Your" so that we see:

```
docker doesn't have enough CPUs
```

instead of

```
Your docker doensn't have enough CPUs
```

Reply:
I'd also include that we require 2 here

```
docker doesn't have access to 2 CPUs, which is the minimum required for minikube
```

Reply:
I was thinking if user specifies more than 2 like 8 the solution message would no be relevent but I could improve by adding another if it is 2